#ifndef __SPH_H__
#define __SPH_H__

#include <ros/ros.h>
#include <nav_msgs/Odometry.h>
#include <ros/topic_manager.h>
#include <std_msgs/String.h>
#include <regex>
#include <map>

#include "water_swarm/Odom.h"
#include "water_swarm/OdomWithNeighbors.h"
#include "water_swarm/OdomBroadcast.h"


#endif